### DISCUSSION TITLE
Support for python-option `-OO`

### DISCUSSION THREAD

**Discussion by mamaraddio** [pyinstaller/pyinstaller#7345] [Features & Development]
I don't really know if this is pyinstaller default operating mode, but I'm thinking that could be great if pyinstaller could support the flag
```
--python-option -OO
```

Do you think it's a possible feature improvement?

**Comment by bwoodsend** [comment]
It used to be a feature that you could run `python -OO -m PyInstaller your-code.py` and it would give you an `-OO` optimized application but pycparser (an indirect dependency of PyInstaller on Windows) now uses a YACC implementation which is written in functions' docstrings (don't ask me why) so it's now unusable. It was such a micro optimization though - removing asserts never speeds anything up noticeably and removing docstrings saved about 10kB on my 100MB application.
